src/target_intacct/const.py

```diff
@@ -1,19 +1,21 @@
 REQUIRED_CONFIG_KEYS = [
-    'company_id',
-    'sender_id',
-    'sender_password',
-    'user_id',
-    'user_password',
+    "company_id",
+    "sender_id",
+    "sender_password",
+    "user_id",
+    "user_password",
+    "object_name",
+    "entity_id",
 ]
 
 KEY_PROPERTIES = {
```

Comment:
Is this variable used anywhere now that we've removed the import? Any time you remove the last reference to a thing you should also probably remove the thing.